Add files via upload - New theme JS files

WebCodePro719 opened the pull request:
I've created a fork of the ECharts master, and uploaded new theme JS files to the fork. I've also included the theme screenshots for users who may not go to the ECharts website to look at the themes. 

I've created this pull request per this post: https://github.com/apache/incubator-echarts/issues/11550

I hope I've completed the pull request correctly. If I need to do something else to contribute the custom theme files to ECharts please let me know. Thanks!

pissang commented:
Thanks a lot!

WebCodePro719 commented:
You're welcome, and thanks for adding the themes!

pissang commented on theme/azul.js:
```diff
@@ -0,0 +1,209 @@
+        }
+    },
+
+	k: {
```
`k` has been renamed to `candlestick` since 3.0.0

WebCodePro719 replied:
Ok, see post on main thread. Thanks!

pissang commented on theme/azul.js:
```diff
@@ -0,0 +1,209 @@
+                    color0: '#f5a503'   // Yinbian border color
+                }
+            }
+        }
```
The hierarchy of `itemStyle` since 4.0.0 has been changed to

```js
itemStyle: {
   color: ...
},
lineStyle: {
   color: ...
},
areaStyle: {
   color: ...
}
```

WebCodePro719 replied:
I'm a little confused about this. Please see post on main thread. Thanks!

pissang commented on theme/azul.js:
```diff
@@ -0,0 +1,209 @@
+        }
+    },
+
+	force : { 
```
`force` has been ranamed to `graph` since 3.0.0

WebCodePro719 replied:
Ok, see post on main thread. Thanks!

WebCodePro719 commented:
@pissang I noticed that you requested changes to the Azul theme (renamed a few elements, etc.) Most of the themes will probably need these updates. 

Would you like me to go through and update all the theme files and then do another pull request?

There's a lot of files, and I don't want you or other contributors to have to do the work. 

Also, if you see any other lines of code in the themes that need to be changed, please let me know.

Thanks!

WebCodePro719 commented:
@pissang I've updated all the theme JS files so that "k" is renamed as "candlestick" and "force" is renamed as "graph." 

For the hierarchy of itemStyle, are you saying that "normal: {....}" should be removed?

For example, in azul,js, would this code be the correct?:

`candlestick: {
	itemStyle: {
                 color: '#f2385a',
                 color0: '#f5a503',
	},
	lineStyle: {
                 width: 1,
                 color: '#f2385a',
                 color0: '#f5a503',
	},
	areaStyle: {
                 color: '#eee',
                 color0: '#ddd'
	}
},

graph: { 
	itemStyle: {
                 color: '#f2385a',
	},
        linkStyle: {
                 color: '#f2385a'
        }
},


map: { 
	itemStyle: {
                 color: '#f2385a',
	},
	areaStyle: {
                 color: '#eee',
                 label: {
                    textStyle: {
                        color: '#c12e34'
                    }
              }
	}
},`

Please let me know if I've update the code correctly in the above example. If so, I'll go through and update all the theme JS files to reflect these code changes.

If you see anything else that needs to be fixed (based on azul.js). please let me know. Thanks!

pissang commented:
Hi @WebCodePro719 

```js
candlestick: {
	itemStyle: {
                 color: '#f2385a',
                 color0: '#f5a503',
	},
	lineStyle: {
                 width: 1,
                 color: '#f2385a',
                 color0: '#f5a503',
	},
	areaStyle: {
                 color: '#eee',
                 color0: '#ddd'
	}
},

graph: { 
	itemStyle: {
                 color: '#f2385a',
	},
        linkStyle: {
                 color: '#f2385a'
        }
}
```
These are correct. `normal` is removed

```
map: { 
	itemStyle: {
                 color: '#f2385a',
	},
	areaStyle: {
                 color: '#eee',
                 label: {
                    textStyle: {
                        color: '#c12e34'
                    }
              }
	}
},
```

Labe should be in the same level with `areaStyle` and `itemStyle`. Also `textStyle` is removed. So it sould be:
map: { 
	itemStyle: {
                 color: '#f2385a',
	},
	areaStyle: {
                 color: '#eee'
	},
        label: {
              color: '#c12e34'
        }
},
```

All these changes since 4.0 are trying to make `option` more flatten and cleaner.

WebCodePro719 commented:
Hi @pissang - Thank you for your help, I really appreciate it! I'll go through and fix all the theme files. I'll upload the fixed files to GitHub as soon as I can. It will probably take me a few days.

WebCodePro719 commented:
Hi @pissang  - I've updated the code in all the theme JS files, including five of the default themes.

Some of the themes were created by other users and contributed to ECharts back when ECharts was version 2.0. For these themes, I changed as much of the code as I could. I'm not sure how to update some of the code in the older 2.0 themes.

I've uploaded all the updated files to the forked repository. If I need to do anything else, please let me know.
